cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -47,7 +47,7 @@ virsh destroy $DOMAIN_NAME || true
 virt-sysprep -d $DOMAIN_NAME --operations machine-id,bash-history,logfiles,tmp-files,net-hostname,net-hwaddr
 
 # Remove VM
-virsh undefine $DOMAIN_NAME
+virsh undefine $DOMAIN_NAME --nvram
```

Comment:
Why is it necessary to pass --nvram? isnt it enough to use `virsh undefine` as is?

Reply:
`virsh undefine` fails without it on aarch64